Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM redhat/ubi9-minimal:9.2
+
+WORKDIR /app/qpc
+VOLUME ["/root/.config/qpc"]
+VOLUME ["/root/.local/share/qpc"]
+
+COPY pyproject.toml poetry.lock ./
+RUN microdnf update -y \
+    && microdnf install -y \
+        git \
+        jq \
+        make \
+        man-db \
+        man \
+        openssh-clients \
+        procps-ng \
+        python3.11 \
+        python3.11-pip \
+        tar \
+        which \
+    && if [[ ! -e /usr/bin/python ]]; then ln -sf /usr/bin/python3.11 /usr/bin/python; fi \
+    && if [[ ! -e /usr/bin/pip    ]]; then ln -sf /usr/bin/pip3.11    /usr/bin/pip;    fi \
+    && pip install -U pip \
+    && pip install poetry \
+    && poetry config virtualenvs.in-project true \
+    && poetry install -n
+
+ENV VIRTUAL_ENV=/app/qpc/.venv
+ENV PATH="${VIRTUAL_ENV}/bin:$PATH"
+
+# Copy QPC
+COPY . .
+COPY docs/qpc.1 /usr/local/share/man/man1/qpc.1
```

Comment:
Aren't we missing a `make manpage` before copying the file here?

Reply:
It's in the Makefile dependency for the build-container target.

```
build-container: manpage
  podman build ....
```

Reply:
aaah I did miss that part! cool